Add invisible FARP

vhirtham opened the pull request:
Hi, I started to write a new mission tool using `pydcs` and couldn't find the new invisible FARP. So I added it. I also added a new `farp_type` parameter to `Mission.farp`. This lets you change the farp type by passing a class reference. I moved it to the end so that it hopefully doesn't cause any conflicts with the existing code.

Here is a small sample script that generates a mission with one instance of each farp type:

~~~ python
import dcs

mission = dcs.Mission(dcs.terrain.Syria())

hatay = mission.terrain.hatay()

usa = mission.country("USA")
mission.farp(country=usa,
             name="FARP 1",
             position=dcs.Point(hatay.position.x + 1000, hatay.position.y + 1000),
             )

mission.farp(country=usa,
             name="FARP 2",
             position=dcs.Point(hatay.position.x + 2000, hatay.position.y + 1000),
             farp_type=dcs.unit.SingleHeliPad
             )

mission.farp(country=usa,
             name="FARP 3",
             position=dcs.Point(hatay.position.x + 3000, hatay.position.y + 1000),
             farp_type=dcs.unit.InvisibleFARP
             )


mission.save("farp_sample.miz")
~~~

Let me know what you think.

rp- commented:
I think it would be better to have a list of possible FARP types and also check that any user didn't specify an invalid class type on the last parameter.

vhirtham commented:
Thanks for the feedback. I will probably update the branch on Wednesday (adding your suggestions and fixing the workflows).

vhirtham commented:
I updated the branch. Flake8 issues should be fixed and the FARP type is checked to be valid. Additionally, I added the option to pass the name of the FARP instead of the type.

vhirtham commented:
Missed one flake8 issue, due to all the "line to long" warnings because I was too lazy to configure my system accordingly. Fixed it.

rp- commented:
Thanks
